### ISSUE TITLE
Cannot reproduct your experiment results?

### ISSUE THREAD

**Issue by smallQ-Q** [IC3Net/IC3Net#13]
Hi prefessor :) Thank you for making a great contribution to MARL areas and your gate mechanism plays an important role in communication. Also it improves the communication efficiency.
Here I meet a tiny problem that when I run 
`python main.py --env_name predator_prey --nagents 3 --nprocesses 16 --num_epochs 2000 --hid_size 128 --detach_gap 10 --lrate 0.001 --dim 5 --max_steps 20 --ic3net --vision 0 --recurrent
`
in PP env with easy and hard mode, the results of average steps-taken are 15 and 75 respectively, which is differnt from you result of 8.9 and 52.4.
I can confirm that I don't update any code. So, could you kindly tell me why it happend?
